projects/FinalProject-Bugs/dominion/unittest10.c

```diff
@@ -0,0 +1,103 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include "dominion.h"
+#include <string.h>
+#include "rngs.h"
+#include "dominion_helpers.h"
+
+int main()
+{
+	int i, r;
+	int currentPlayer = 0;
+	int discardCount;
+	int handCount;
+	int k[10] = { ambassador, council_room, feast, gardens, mine, remodel, smithy, village, treasure_map, great_hall };
+	int maps;
+	int nextPlayer = 1;
+	int numPlayers = 2;
+	int seed = 1000;
+	struct gameState G;
+
+	memset(&G, 23, sizeof(struct gameState)); //clear the game state
```

Comment:
Didn't set this in @gibsojen's tests. Should we go back and add them?

Reply:
Clearing the game state?  I can do that.